variants/bullet/template.rb

```diff
@@ -1,13 +1,13 @@
 insert_into_file "Gemfile", after: /^group :development, :test do\n/ do
-<<-GEMS
+  <<-GEMS
```

Comment:
maybe a `<<~` should go here? 🐘

Reply:
Some of the usage of `<<-` is deliberate to achive a particular indentation. If all the lines are the same indentation then `<<~` will remove all prefixed whitespace.

Some of the usage is just because `<<~` didn't exist when that code was written so could probably be upgraded. It's a small enough win that I'm not keen to spend time on the effort.